example_test.go

```diff
@@ -0,0 +1,40 @@
+package quad_test
+
+import (
+	"fmt"
+	"math"
+	"runtime"
+
+	"github.com/gonum/quad"
+	"github.com/gonum/stat/dist"
+)
+
+func Example() {
+	fmt.Println("Evaluate the expected value of x^2 + 3 under a Weibull distribution")
+	f := func(x float64) float64 {
+		d := dist.Weibull{Lambda: 1, K: 1.5}
+		return (x*x + 3) * d.Prob(x)
+	}
+	ev := quad.Fixed(f, 0, math.Inf(1), 10, nil, 0)
+	fmt.Printf("EV with 10 points = %0.6v\n", ev)
+	ev = quad.Fixed(f, 0, math.Inf(1), 30, nil, 0)
+	fmt.Printf("EV with 30 points = %0.6v\n", ev)
+	ev = quad.Fixed(f, 0, math.Inf(1), 100, nil, 0)
+	fmt.Printf("EV with 100 points = %0.6v\n", ev)
+	ev = quad.Fixed(f, 0, math.Inf(1), 10000, nil, 0)
+	fmt.Printf("EV with 10000 points = %0.6v\n", ev)
+	fmt.Println()
```

Comment:
I thought it was easier to read, and nice to highlight a different action was being taken (not just larger quadrature points).